docs/ruleset_functions/G311_exceptions/G311_sub_functions/is_space_a_computer_room.md

```diff
@@ -1,31 +1,31 @@
 # is_space_a_computer_room
 
-**Description:** Returns true or false as to whether space is a computer room. The criteria is such that it is considered a computer room if the lighting space type definition equals COMPUTER_ROOM, or, if the space type is not defined, if the total misc W/sf exceeds 20 W/sf per the definition of a computer room in 90.1 Section 3. 
+**Description:** Returns true or false as to whether space is a computer room. The criteria is such that it is considered a computer room if the total of misc INFORMATION_TECHNOLOGY_EQUIPMENT Power density in W/sf exceeds 20 W/sf per the definition of a computer room in 90.1 Section 3.  
 
 **Inputs:**  
 - **B-RMI,P-RMI**: The applicable ruleset model instance.   
 - **Space_obj**: The space to assess whether or not it is a computer room.
 
 **Returns:**  
-- **is_space_a_computer_room**: The function returns true or false as to whether space is a computer room. The criteria is such that it is considered a computer room if the lighting space type definition equals COMPUTER_ROOM, or, if the space type is not defined, if the total misc W/sf exceeds 20 W/sf per the definition of a computer room in 90.1 Section 3. 
+- **is_space_a_computer_room**: The function returns true or false as to whether space is a computer room. The criteria is such that it is considered a computer room if the total of misc INFORMATION_TECHNOLOGY_EQUIPMENT Power density in W/sf exceeds 20 W/sf per the definition of a computer room in 90.1 Section 3. 
+ 
 
 **Function Call:**  None  
 
 
 ## Logic:    
-- Set is_space_a_computer_room to false: `is_space_a_computer_room = false`  
-- Check if the lighting space type classification equals computer room: `if space_p.lighting_space_type == "COMPUTER_ROOM": is_space_a_computer_room = true`
-- Else, carry on with logic: `Else:`                 
-    - Reset the total_space_misc_Wattage_including_multiplier variable: `total_space_misc_Wattage_including_multiplier = 0`
-    - For each miscellaneous equipment id in the space: `For misc_p in space_p.miscellaneous_equipment:`
-        - Check if the energy type is electricity, if not then skip this misc_p: `if misc_p.energy_type = "ELECTRICITY":`
+- Set is_space_a_computer_room to false: `is_space_a_computer_room = false`               
+- Set the total_space_misc_Wattage_including_multiplier variable: `total_space_misc_Wattage_including_multiplier = 0`
+- For each miscellaneous equipment id in the space: `For misc_p in space_p.miscellaneous_equipment:`
+    - Check if the energy type is electricity, if not then skip this misc_p: `if misc_p.energy_type = "ELECTRICITY":`
+        - Check if the energy type is INFORMATION_TECHNOLOGY_EQUIPMENT: `if misc_p.type == "INFORMATION_TECHNOLOGY_EQUIPMENT":`
```

Comment:
I am goign to approve this but just documenting my concern about requiring this to be categorized thereby creating another input that is will need to bee addressed outside of the model; most tools do not classify to this granularity. The majority ruled against me but just wanted to document this.

Reply:
To address the concern, I think this should be raised in a general guideline for BEM vendor or included in the test cases so that vendors know how to interpret misc equipment to be applicable to computer rooms.
I expect most of the work is done by the vendors so that modelers can just follow the vendor's workflow to have the correct RPD file generated.